<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearningTab</class>
 <widget class="QWidget" name="LearningTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="NetworkFileLoad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>NetworkFileLoad</string>
   </property>
  </widget>
  <widget class="QLabel" name="NeuronsDeletePercent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>23</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NeuronsDeletePercent</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Learn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Learn</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ScenarioFileLoad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ScenarioFileLoad</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NetworkFileText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>211</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScenarioFileText">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>241</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NeuronsCreatePercent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NeuronsCreatePercent</string>
   </property>
  </widget>
  <widget class="QLabel" name="SynapsesDeletePercent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>74</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SynapsesDeletePercent</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SynapsesCreatePercentValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>95</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="SynapsesCreatePercent">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SynapsesCreatePercent</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SynapsesDeletePercentValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="MutationOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>124</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MutationOutput</string>
   </property>
  </widget>
  <widget class="QLabel" name="SelectionOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>SelectionOutput</string>
   </property>
  </widget>
  <widget class="QLabel" name="EpochsLimit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>173</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>EpochsLimit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="EpochsLimitValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>168</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="SelectionOutputValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>144</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MutationOutputValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NeuronsCreatePercentValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>45</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="NeuronsDeletePercentValue">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
